src/components/organisms/BottomPaneManager/BottomPaneManager.tsx

```diff
@@ -33,6 +33,7 @@ const BottomPaneManager: React.FC = () => {
   const settings = useAppSelector(state => state.terminal.settings);
   const shellsMap = useAppSelector(state => state.terminal.shellsMap);
   const terminalsMap = useAppSelector(state => state.terminal.terminalsMap);
+  const terminalHeight = useAppSelector(state => state.terminal.height);
```

Comment:
We have the `bottomPaneHeight` from the pane configuration. We could maybe use that instead for fixing this

Reply:
done! It is better than before, thank you @topliceanurazvan